### ISSUE TITLE
Crash after opening Preferences

### ISSUE THREAD

**Issue by campos20** [xournalpp/xournalpp#1951]
**Affects versions :**
 - OS: Ubuntu 20.04
 - (Linux only) Desktop environment: Gnome
 - Which version of libgtk do you use: 3.24.18-1ubuntu1
 - Version of Xournal++: 1.0.18, commit 535d261d
 - Installation method: official release

**Describe the bug**
Complete crash after opening preferences.

**To Reproduce**
Open terminal, type `xournalpp`. Go to Edit/Preferences.

**Expected behavior**
Open preferences Window. Change settings.

**Screenshots of Problem**
Actually, paste from terminal.
```
lexandre@alexandre-note:~$ xournalpp
ALSA lib pcm.c:2642:(snd_pcm_open_noupdate) Unknown PCM cards.pcm.rear
ALSA lib pcm.c:2642:(snd_pcm_open_noupdate) Unknown PCM cards.pcm.center_lfe
ALSA lib pcm.c:2642:(snd_pcm_open_noupdate) Unknown PCM cards.pcm.side
ALSA lib pcm_route.c:869:(find_matching_chmap) Found no matching channel map
ALSA lib pcm_route.c:869:(find_matching_chmap) Found no matching channel map
ALSA lib pcm_route.c:869:(find_matching_chmap) Found no matching channel map
ALSA lib pcm_route.c:869:(find_matching_chmap) Found no matching channel map
Cannot connect to server socket err = Arquivo ou diretório inexistente
Cannot connect to server request channel
jack server is not running or cannot be started
JackShmReadWritePtr::~JackShmReadWritePtr - Init not done for -1, skipping unlock
JackShmReadWritePtr::~JackShmReadWritePtr - Init not done for -1, skipping unlock
Cannot connect to server socket err = Arquivo ou diretório inexistente
Cannot connect to server request channel
jack server is not running or cannot be started
JackShmReadWritePtr::~JackShmReadWritePtr - Init not done for -1, skipping unlock
JackShmReadWritePtr::~JackShmReadWritePtr - Init not done for -1, skipping unlock
ALSA lib pcm_oss.c:377:(_snd_pcm_oss_open) Unknown field port
ALSA lib pcm_oss.c:377:(_snd_pcm_oss_open) Unknown field port
ALSA lib pcm_usb_stream.c:486:(_snd_pcm_usb_stream_open) Invalid type for card
ALSA lib pcm_usb_stream.c:486:(_snd_pcm_usb_stream_open) Invalid type for card
Cannot connect to server socket err = Arquivo ou diretório inexistente
Cannot connect to server request channel
jack server is not running or cannot be started
JackShmReadWritePtr::~JackShmReadWritePtr - Init not done for -1, skipping unlock
JackShmReadWritePtr::~JackShmReadWritePtr - Init not done for -1, skipping unlock

(xournalpp:14421): Gtk-WARNING **: 18:14:18.541: Theme parsing error: <data>:1:22: Junk at end of value for font-family
xournalpp: symbol lookup error: xournalpp: undefined symbol: _ZN9portaudio6System14DeviceIteratorneERKS1_
alexandre@alexandre-note:~$ 

```

**Additional context**
I had to manually install a lot of dependencies.

**Comment by Febbe** [comment]
The official releases are binary incompatible with ubuntu20.04, please use the ppa and report back. Also don't open a ticket before you checked the issue history. There are quite a lot duplicates of this one.

**Comment by campos20** [comment]
Ppa worked. Sorry for the inconvenience.
